static/yarn.lock

```diff
@@ -2099,8 +2099,8 @@ electron-winstaller@^5.0.0:
 
 electron@*, electron@20.3.8:
   version "20.3.8"
-  resolved "https://registry.yarnpkg.com/electron/-/electron-20.3.11.tgz#dd3a1a26edecd012a807ae96409d52d0a0cc4f11"
-  integrity sha512-+dZRzUCDIeVcOqha4hW/y/rNOjoM23rcuLQgrdzayj2FHfgs8mud6fAzvTeJN3TT2c6em/u5cZYQXqdrYcZqAg==
+  resolved "https://registry.yarnpkg.com/electron/-/electron-20.3.8.tgz#94a4b2bb89a3e6bbeb824365cfd0f65658bc1a2c"
```

Comment:
Introduced in https://github.com/logseq/logseq/pull/8406/files#diff-86f7aa91241d22d5a79d93bd13a984edf1101b7ead21657fbedf9f2721c8ec7c